### DISCUSSION TITLE
How can I disable SSL in ZooKeeper?

### DISCUSSION THREAD

**Discussion by runkecheng** [strimzi/strimzi-kafka-operator#9148] [Q&A] [answered]
Properties with the prefixe `ssl.` cannot be set

However, some third-party ui tools(like offset explorer) do not support the use of ssl when connecting to zk.

**Comment by scholzj** [comment, marked as answer]
You cannot disable SSL in ZooKeeper and no tools should normally need it for anything. ZooKeeper is going away from Kafka, so tools needing ZooKeeper will stop working. There are also no offsets stored in ZooKeeper, so not sure why would any offset exprorer actually need it.

If you want, you can use this unofficial tool to expose ZooKeeper without SSL: https://github.com/scholzj/zoo-entrance ... but it is completely insecure, so use it at your own risk.
